### PR TITLE
Form for AddWork page. Edit title in header.

### PR DESCRIPTION
**Pull request by BaraT91**

### CODE REVIEW

File: src/Pages/AddWork/work.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.work-page {
```
Comment (Tondakozak) [inline src/Pages/AddWork/work.css:1]: Suggestion:
maybe rather "Log Work Page"?

File: src/Pages/AddWork/work.tsx [hunk 2]
```diff
@@ -1,3 +1,78 @@
-import './work.css'
+import React from 'react';
+import './work.css';
 
-export const Work: React.FC = () => <></>;
+export const Work: React.FC = () => {
+  return (
+    <div className="work-page">
```
Comment (Tondakozak) [inline src/Pages/AddWork/work.tsx:6]: It would be better to make React components.

File: src/index.css [hunk 3]
```diff
@@ -25,24 +25,15 @@ a:hover {
 
 body {
```
Comment (Tondakozak) [inline src/index.css:26]: The page content should be being added into a div with max-width, I guess.

**Review by Tondakozak — APPROVED** [review]
Good job, a few notes for the future.

File: src/Pages/AddWork/work.css (hunk 1 above)
Comment (BaraT91) [reply src/Pages/AddWork/work.css:1]: I will rename it

File: src/Pages/AddWork/work.tsx (hunk 2 above)
Comment (BaraT91) [reply src/Pages/AddWork/work.tsx:6]: Sure. Tomorrow 😇

**Pull request merged**